api/loader/fileloader.go

```diff
@@ -164,30 +164,49 @@ func (fl *fileLoader) New(path string) (ifc.Loader, error) {
 	if path == "" {
 		return nil, fmt.Errorf("new root cannot be empty")
 	}
-	repoSpec, err := git.NewRepoSpecFromUrl(path)
-	if err == nil {
-		// Treat this as git repo clone request.
+
+	root, errD := func() (filesys.ConfirmedDir, error) {
```

Comment:
The previous order is first to treat target as a remote target; when failed, treat it as a local target. The changes make the order reverted. Is there any reason for doing this?

Reply:
In fact, getter is capable to handle local target so we cannot check it with the error return as `repoSpec, err := git.NewRepoSpecFromUrl(path)`. However it may not behavior exactly the same, e.g. the containment violation. 

Putting local target before getter is to ensure backward compatibility.